### ISSUE TITLE
Simplify using field background

### ISSUE THREAD

**Issue by sbastrakov** [ComputationalRadiationPhysics/picongpu#2944]
If I understand correctly, in case a custom field background is used, all 3 field backgrounds have to be set in `fieldBackground.param`. Since only 1 or 2 may be actually needed, I suggest the following simplification. We provide a dummy background class following the requirements and being disabled or always returning zero. Then in case, e.g., only J background is needed, a user can write in her `fieldBackground.param`:
```
using FieldBackgroundE = [our new class];
using FieldBackgroundB = [our new class];
class FieldBackgroundJ {
   [proper implementation]
};
```
and thus have a small simplification.

I've encountered this scenario during preparation of absorber test example.

Edit: in case that is okay, I can provide a PR.

**Comment by ax3l** [comment]
Sounds good as long as users find an easy example to copy from.
We kept the zero-cases explicitly in so they are easier to modify.

**Comment by PrometheusPi** [comment]
I agree with @ax3l - a consistent setup as suggested by @sbastrakov would avoid accidentally having a additionally (wrong) background field.
